verilog/CST/expression.cc

```diff
@@ -98,6 +98,19 @@ const verible::Symbol* GetConditionExpressionFalseCase(
   return GetSubtreeAsSymbol(condition_expr, NodeEnum::kConditionExpression, 4);
 }
 
+const verible::TokenInfo* GetUnaryPrefixOperator(
+    const verible::Symbol& symbol) {
+  const auto& children = verible::SymbolCastToNode(symbol).children();
```

Comment:
if symbol does not happen to be a node, then this will crash. You probably want to return `nullptr` in case `symbol.Kind()` is not a node.
Or you could test that it is a `kUnaryPrefixExpression`, which is a better, more specific, test.

(Also add a test for it to see that other nodes than kUnaryPrefixExpressions always return nullptr).

Reply:
Added the checks in both functions, thanks pointing it out.